pages/glossary/content-that-becomes-visible.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Content that becomes visible
+key: content-that-becomes-visible
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The _content that becomes visible_ after an [event][] [firing][] is any element for which all of the following is true:
+
+- 500 milliseconds after the [event][] [firing][] the element has [visible text content][]; and
+- a tick before the [event][] [firing][] there was no element with the same [descendant text content][] that the element has after the [event][] [firing][]; and
```

Comment:
Seems to me like this would apply to elements that update their content too. If I have a button that changes its text on hover, it 1. has visible content after the event. and 2. does not have the same descendant text content. Additionally, descendant text content does not take shadow DOM into account.